### ISSUE TITLE
Missing sounds warning

### ISSUE THREAD

**Issue by ISmileEveryday** [HaloOfBlocks/AdditionalGuns#85]
hey all. I made a server for me and my friends and one of them pointed out an issue he had with missing sounds or something

heres the lines he got:
```
[Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: additionalguns:item.1911.fire
[Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: additionalguns:item.1911.suppressed
[Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: additionalguns:item.1911.enchanted
```

is there something he needs to do or can this get fixed?
im still new to modded minecraft so dont know some of these things, sorry about that

**Comment by CasualNoLifer** [comment]
> hey all. I made a server for me and my friends and one of them pointed out an issue he had with missing sounds or something
> 
> heres the lines he got:
> 
> ```
> [Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: additionalguns:item.1911.fire
> [Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: additionalguns:item.1911.suppressed
> [Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: additionalguns:item.1911.enchanted
> ```
> 
> is there something he needs to do or can this get fixed? im still new to modded minecraft so dont know some of these things, sorry about that

Hi there. The M1911 and sounds came in a newer update (0.7.0). Did you make sure that you and your friends are all using the same version of Additional Guns?

**Comment by ISmileEveryday** [comment]
> Hi there. The M1911 and sounds came in a newer update (0.7.0). Did you make sure that you and your friends are all using the same version of Additional Guns?

yes we all use 0.7.2 

were on 1.18.2 if that helps

**Comment by CasualNoLifer** [comment]
Would you be willing to have him test some of the other guns? The Magnum in particular since it came out in the same update as the M1911.

**Comment by Autovw** [comment]
Verified the issue on my end. It's just a message printed in the log upon startup. It's not something to worry about. We'll look into fixing this (if possible) in the very next update.

Thanks for reporting!

**Comment by ISmileEveryday** [comment]
thanks for the quick response again!

glad to know its nothing serious

**Comment by Autovw** [comment]
Issue is fixed in Additional Guns 0.8.0.
Thanks for reporting!
